sql_generators/glean_usage/__init__.py

```diff
@@ -103,7 +153,7 @@ def generate(target_project, output_dir, parallelism, exclude, only, app_name):
     if app_name:
         app_info = {name: info for name, info in app_info.items() if name == app_name}
 
-    app_info = [info for name, info in app_info.items() if name not in SKIP_APPS]
+    app_info = [info for name, info in app_info.items() if name in ALLOWED_APPS]
```

Comment:
Do we know if the baseline tables are all deploying correctly automatically? If it's specifically metrics ETL that has been causing issues, then perhaps we can still allow all the baseline ping ETL to flow for new apps?

Reply:
Right, baseline tables were unaffected by this. I'll make some updates